tchannel/thrift/generated.py

```diff
@@ -0,0 +1,95 @@
+from __future__ import (
+    absolute_import, division, print_function, unicode_literals
+)
+
+import inspect
+import types
+
+from .util import get_service_methods
+
+
+def from_thrift_module(service, thrift_module):
```

Comment:
There's room for confusion between the modules and the schemes though:

```
import tchannel.thrift
vs.
tchannel = TChannel()
response = tchannel.thrfit(...)
```

Personally I think it would be worth re-visiting the library managing the TChannel singleton for you (kind of like how Tornado keeps track of the IOLoop but still exposes enough control for power users). Then you could just interact with modules like `tchannel.thrift` which would use `TChannel` behind the scenes.